BeardedSpice/BSTrack.m

```diff
@@ -18,19 +18,21 @@
 NSString *const kBSTrackNameFavorited = @"favorited";
 NSString *const kBSTrackNameIdentifier = @"BSTrack Notification";
 
+const NSString *kImageLock = @"kImageLock";
+
 @interface BSTrack ()
 
 @property (nonatomic, strong) NSString *imageURL;
 
-@property (nonatomic, strong) NSString *lastImageUrlString;
-@property (nonatomic, strong) NSImage *lastImage;
-
 - (void)_loadImage;
 
 @end
 
 @implementation BSTrack
 
+static NSString *_lastImageUrlString;
+static NSImage *_lastImage;
```

Comment:
"This makes them singleton values" - this is what i mean. This should be the case.